AdminWebsite/AdminWebsite.UnitTests/Controllers/ReferenceDataControllerTests.cs

```diff
@@ -143,42 +146,48 @@ private List<CaseTypeResponse> GetCaseTypesList()
         {
             return new List<CaseTypeResponse>
             {
-                new CaseTypeResponse
+                new ()
                 {
-                    Id = 1, Name = "type1",
+                    Id = 1, Name = "type1", ServiceId = "AA1",
                     HearingTypes = new List<HearingTypeResponse>()
                     {
-                        new HearingTypeResponse() {Id = 10, Name = "HType10"},
+                        new () {Id = 10, Name = "HType10", Code = "Code10"},
                     }
                 },
-                new CaseTypeResponse
+                new()
                 {
-                    Id = 2, Name = "type2",
+                    Id = 2, Name = "type2", ServiceId = "AA2",
                     HearingTypes = new List<HearingTypeResponse>()
                     {
-                        new HearingTypeResponse() {Id = 20, Name = "HType20"},
+                        new () {Id = 20, Name = "HType20", Code = "Code20"},
                     }
                 },
-                new CaseTypeResponse
+                new()
                 {
-                    Id = 3, Name = "type3",
-                     HearingTypes = new List<HearingTypeResponse>()
+                    Id = 3, Name = "type3", ServiceId = "AA3",
+                    HearingTypes = new List<HearingTypeResponse>()
                     {
-                        new HearingTypeResponse() {Id = 25, Name = "HType25"},
-                        new HearingTypeResponse() {Id = 29, Name = "HType29"},
+                        new () {Id = 25, Name = "HType25", Code = "Code25"},
+                        new () {Id = 29, Name = "HType29", Code = "Code29"},
                     }
+                },
+                new()
+                {
+                    Id = 4, Name = "type4", ServiceId = "AA4",
+                    HearingTypes = new List<HearingTypeResponse>()
                 }
             };
         }
 
-        private List<HearingTypeResponse> GetHearingTypes()
+        private List<AdminWebsite.Contracts.Responses.HearingTypeResponse> GetHearingTypes()
```

Comment:
could we make this method static to remove the sonar cloud warning?

Reply:
Yeah, sure thing